### DISCUSSION TITLE
Question about interactive menus

### DISCUSSION THREAD

**Discussion by LDprg** [Samillion/ModernZ#214] [Q&A] [answered]
First thx for maintaining this project. 
I got a problem with interactive menus, they are not interactive at all (probably my fault).
I use the default config of modernZ. The problem is that when selecting Audio/Subtitles/Playlist, the selector works with keyboard only and is behind the OSC (which is annoying since every time I move my mouse it is displayed over the selector).  Is there extra configuration needed for enabling mouse/touch support? And is it possible to move the menu above the OSC?
Thx, in advance!

**Comment by Samillion** [comment, marked as answer]
The behavior you're describing sounds like outdated mpv/scripts.

If you don't update mpv, then alternatively you can manually use the updated [console.lua](https://github.com/mpv-player/mpv/blob/master/player/lua/console.lua) and [select.lua](https://github.com/mpv-player/mpv/blob/master/player/lua/select.lua), which include full mouse support.

Though I highly recommend updating mpv to the latest, many improvements have been introduced recently.

**Comment by LDprg** [comment]
Ohhhh got it, thx. 
For every other Arch linux user that has the same problem: Install mpv-git instead of mpv!
